### ISSUE TITLE
May be a bug in the RunUAT.sh script?

### ISSUE THREAD

**Issue by itislee** [adamrehn/ue4-docker#206]
Output of the `ue4-docker info` command:

```
ue4-docker version:         0.0.91 (latest available version is 0.0.91)
Operating system:           Linux 
Docker daemon version:      19.03.1
NVIDIA Docker supported:    No
Maximum image size:         No limit detected
Available disk space:       97.16 GiB
Total system memory:        15.42 GiB physical, 0 bytes virtual
Number of processors:       8 physical, 8 logical
```

Additional details:

- Are you accessing the network through a proxy server?  NO

I'am building ue4.26.0 using command:
/usr/local/bin/ue4-docker build 4.26.0 --exclude debug --exclude templates --monitor -interval=10

and everything goes well and got the image,but when using the image something goes wrong.Details as below :
```
Running AutomationTool...

/home/ue4/UnrealEngine/Engine/Build/BatchFiles/RunUAT.sh: line 65: /home/ue4/UnrealEngine/Engine/Build/BatchFiles/Mac/SetupEnvironment.sh: No such file or directory
Start UAT: mono AutomationTool.exe BuildCookRun -utf8output -clientconfig=Shipping -serverconfig=Shipping -project=/tmp/project/NTDemo.uproject -noP4 -cook -build -stage -prereqs -pak -archive -server -noclient -platform=Linux -serverplatform=Linux -nodebuginfo -archivedirectory=/tmp/project/dist
env: 'mono': No such file or directory
RunUAT ERROR: AutomationTool was unable to run successfully.
```
Looking into  /home/ue4/UnrealEngine/Engine/Build/BatchFiles/Mac/SetupEnvironment.sh, I found this:
if [ "$(uname)" = "Linux" ]; then
        # Setup Environment and Mono
        source "$SCRIPT_DIR/Mac/SetupEnvironment.sh" -mono "$SCRIPT_DIR/Linux"
fi

Then I try with version 4.27.0, and I found this:
if [ "$(uname)" = "Linux" ]; then
        # Setup Environment and Mono
        source "$SCRIPT_DIR/Linux/SetupEnvironment.sh" -mono "$SCRIPT_DIR/Linux"
fi

**Comment by TBBle** [comment]
Yup, bug in `RunUAT.sh`, fixed [in 4.26.1](https://github.com/EpicGames/UnrealEngine/commit/961bc4247f93c6538b8c97fd65b60f83028a52a4).

**Comment by TBBle** [comment]
I'm closing this, since it was a UE upstream issue fixed in a later patch release, so probably not worth patching around in ue4-docker.
